install-linux.sh

```diff
@@ -111,5 +141,5 @@ fi
 
 echo ""
 echo "Thank you for supporting Zyn-Fusion"
-echo "You can now use the 3.0.3 release via a LV2/VST plugin host or"
+echo "You can now use the 3.0.3 release via a LV2/VST/VST3/CLAP plugin host or"
```

Comment:
Note to self, I really have to remember to update this version number :p The rest of the changes look reasonable to me.

Reply:
It's better to have a function for fetching version number.